README.md

```diff
@@ -131,6 +136,25 @@ Then follow the steps to [run Atoti UI](https://activeviam.com/activeui/document
 
 Finally open your dashboards, widgets, filters, ... Make sure they work as expected.
 
+## Migrate an Atoti jupyter notebook
+
+To migrate a jupyter notebook, two files are necessary:
+
+- _notebook.ipynb_, the notebook to migrate. It is really important that the notebook has the `.ipynb` extension.
```

Comment:
I'd like to avoid _"really important"_, because I find it too vague. I find _"must"_ enough. Alternatively, we could say something like _"ipynb" is the only supported file extension.

```suggestion
- _notebook.ipynb_, the notebook to migrate. This file must have the `.ipynb` extension.
```